src/renderDigging.js

```diff
@@ -423,14 +423,15 @@ function getPipPoints(state, row, column) {
     const x = column * COLUMN_WIDTH - state.camera.left;
     const y = row * ROW_HEIGHT - state.camera.top + ((column % 2) ? LONG_EDGE : 0);
     const pad = 5;
+    const c = 0.4;
     return [
-        [x + HEX_WIDTH / 2, y + ROW_HEIGHT / 6 + pad * 1.5],
-        [x + SHORT_EDGE + pad, y + ROW_HEIGHT / 3 + pad],
-        [x + HEX_WIDTH - SHORT_EDGE - pad, y + ROW_HEIGHT / 3 + pad],
+        [x + HEX_WIDTH / 4 + HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + 3 * HEX_WIDTH / 4 - HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + SHORT_EDGE, y + ROW_HEIGHT / 2],
```

Comment:
These two center ones should probably use `c` as well, maybe:
`x + HEX_WIDTH / 2 * c`?